### ISSUE TITLE
partyScale: rename to organizationScale

### ISSUE THREAD

**Issue by JachymHercher** [open-contracting/ocds-extensions#165]
https://github.com/open-contracting/standard/issues/1173 proposes that "organization" is preferred to "party", "entity" and similar terms. 

To be consistent, we should rename, in https://extensions.open-contracting.org/en/extensions/partyScale/master/codelists/, "partyScale" to "organizationScale".  

Furthermore, the reference to `parties.details` should be changed, I think, to `organization.details`.

**Comment by jpmckinney** [comment]
For changing the filename of the codelist, @yolile do you know if any publishers patch this codelist?

> Furthermore, the reference to `parties.details` should be changed, I think, to `organization.details`.

`parties` is the name of the field, which cannot be changed in 1.x. `Organization` is the name of the definition. To keep things simple, we typically refer to field names (especially since definition names are allowed to change).

**Comment by JachymHercher** [comment]
> parties is the name of the field, which cannot be changed in 1.x. Organization is the name of the definition. To keep things simple, we typically refer to field names (especially since definition names are allowed to change).

Hmm, I'm probably still a bit lost. Anyway, practically, does the above imply that in https://extensions.open-contracting.org/en/extensions/organizationClassification/1.1/ the `organization.details` should actually be `parties.details`?

**Comment by jpmckinney** [comment]
Well, there Organization is capitalized, so it is at least the correct way to refer to the definition. We should be consistent, but really this has to be a very low priority, across 63 extensions.

**Comment by jpmckinney** [comment]
Well, it turns out we had very few such occurrences (except in extension changelogs, where it's fine to use definition notation, since the changelog is very detailed about schema changes).

I used this regex to find all occurrences `\b[A-Z][a-zA-Z]+\.(?!(aspx|db|html|md|org|xml|zip)\b)[a-zA-Z]{2,}` The long parenthetical is to ignore file extensions (false positives).

All extensions have now been updated. This will be reflected in 24 hours in the Extension Explorer.

I checked and there is no other occurrence of `organization.something` or `Organization.something` (except in changelogs as mentioned).
